tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -353,7 +354,7 @@ StorageManagerCanonical::array_open_for_writes(Array* array) {
 tuple<Status, optional<std::vector<shared_ptr<FragmentMetadata>>>>
 StorageManagerCanonical::array_load_fragments(
     Array* array, const std::vector<TimestampedURI>& fragments_to_load) {
-  auto timer_se = stats_->start_timer("array_load_fragments");
+  auto timer_se = stats_->start_timer("sm_array_load_fragments");
```

Comment:
I'm confused by adding the `sm_` prefix here given we removed it above.

Reply:
So the logic I came up with for the namings (and which you can definitely question as poor :blush: ) is that `array_open_` functions are specific and will be called only from `Array::open` operations/functions, so I wanted to prefix them alike for readability:
```
      "Context.StorageManager.array_open_read_load_schemas_and_fragment_meta.sum": 0.775468,
      "Context.StorageManager.array_open_read_load_schemas_and_fragment_meta.avg": 0.775468,
      "Context.StorageManager.array_open_read_load_directory.sum": 1.4655,
      "Context.StorageManager.array_open_read_load_directory.avg": 1.4655,
      "Context.StorageManager.array_open_READ.sum": 2.2411,
      "Context.StorageManager.array_open_READ.avg": 2.2411,
```

Now, all the rest of `StorageManager` (sm) methods are more generic and can be called from anywhere, some of them are in fact called from `Consolidator` class methods as well. So I decided to continue grouping by prefixing with `sm_`. Also in that logic the general `sm_` stats make up `array_open_` stats if aggregated. And `array_open_` stats make up `array_open_READ/WRITE/..` stats if aggregated.

Please let me know if you have a better ideas on this, I am definitely open.

Reply:
That's enough of a reason for me.